Source/Scene/Cesium3DTileset.js

```diff
@@ -788,19 +835,107 @@ define([
         }
     }
 
-    function getScreenSpaceError(geometricError, tile, frameState) {
+    var scratchPositionNormal = new Cartesian3();
+    var scratchCartographic = new Cartographic();
+    var scratchMatrix = new Matrix4();
+    var scratchCenter = new Cartesian3();
+    var scratchPosition = new Cartesian3();
+    var scratchDirection = new Cartesian3();
+
+    function updateDynamicScreenSpaceError(tileset, frameState) {
+        var up;
+        var direction;
+        var height;
+        var minimumHeight;
+        var maximumHeight;
+
+        var camera = frameState.camera;
+        var root = tileset._root;
+        var tileBoundingVolume = root.contentBoundingVolume;
+
+        if (tileBoundingVolume instanceof TileBoundingRegion) {
```

Comment:
I used this test: https://jsperf.com/instanceof-performance/2.

The code is not much of a hotspot, it is called once per frame.